static/solo-cta/index.js

```diff
@@ -206,13 +206,19 @@ window[cbName] = function (data) {
       }
       
       overlayEl.innerHTML = template;
-
+      var configCopy = JSON.parse(JSON.stringify(config));
+      var onPlayerChange = "onPlayerChange" in config ? 'undefined' !== typeof config.onPlayerChange ? config.onPlayerChange :null : null;
```

Comment:
@luisvelazquex lets simplify this check, you can do coerce

```
var configCopy = JSON.parse(JSON.stringify(config));
configCopy.onPlayerChange = !!configCopy.onPlayerChange
```

https://jsfiddle.net/rdrwnrp4/